client/src/templates/Challenges/classic/show.tsx

```diff
@@ -303,6 +307,19 @@ function ShowClassic({
     setUsingKeyboardInTablist(usingKeyboardInTablist);
   };
 
+  // AB testing Pre-fetch in the Spanish locale
+  const isPreFetchEnabled = useFeature('prefetch_ab_test').on;
+  useEffect(() => {
+    if (
+      isPreFetchEnabled &&
+      !prefetched &&
+      envData.clientLocale === 'espanol'
+    ) {
+      preloadPage(nextChallengePath);
+      setPrefetched(true);
+    }
+  }, [isPreFetchEnabled, nextChallengePath, prefetched]);
```

Comment:
```suggestion
  useEffect(() => {
    preloadPage(nextChallengePath);
  }, [nextChallengePath]);
```
`nextChallengePath` only changes when you navigate to a new page, so this effect triggers once by default.